### ISSUE TITLE
Linked titles don't wrap on mobile

### ISSUE THREAD

**Issue by tomgreever** [donejs/donejs#588]
On the Guides page on mobile, the titles (which are links) don't wrap onto another line and, therefore, extend off the page / device. https://donejs.com/Guides.html

![screen shot 2016-03-24 at 12 04 11 pm](https://cloud.githubusercontent.com/assets/326843/14024788/f9c60378-f1b8-11e5-905e-d53890eab7b5.png)

**Comment by JelmarBosscher** [comment]
Hi, Just came across this.
For anyone bothering fixing this.

In the I assumed compiled static.css theres the following css:

```
body:not(.donejs):not(.community)>.container-fluid .content.docs .comment a,
body:not(.donejs):not(.community)>.container-fluid .content.docs .description a {
    white-space: nowrap;
    font-weight: 900;
    color: #3B83A3;
}
```

I have no knowledge of why this has been done and what changes it may affect or if this is still included because of older code. 

But the `white-space: nowrap` should be removed or overwritten with `white-space: normal`.
This should fix it for this page.

**Comment by matthewp** [comment]
Fixed by #820 

![screen shot 2017-02-15 at 5 22 51 pm](https://cloud.githubusercontent.com/assets/361671/22998131/64627dbe-f3a3-11e6-89f0-920a64fc7217.png)
